### DISCUSSION TITLE
How to match backspace key?

### DISCUSSION THREAD

**Discussion by hargup** [espanso/espanso#1222] [Feature requests and Ideas]
I have to write a lot! And one thing which slows me down is premature editing, I often find myself editing what I am typing in the middle of the typing. To stop that I want to disable consecutive backspace keystrokes. For example, if I press backspace twice, it should be replace with "STOP" or something else. Is it possible with Espanso?

I tried matching with \b, it didn't work.

Basically, I want to force myself to leave typos in your text, so that I'll have to come back later and fix them all at the end.

(Crosspost from reddit  https://www.reddit.com/r/espanso/comments/okruta/matching_backspace_key/h5dbnu1/?context=3)

**Comment by federico-terzi** [comment]
Hey @hargup,

Thank you for the feedback! As I said on Reddit, unfortunately there is no way as of now to do that, as the "backspace" key is treated differently than the others. Things _might_ change in version 2 thanks to the new engine, but I can't guarantee anything :)

What we can do is leaving this issue open and if there is enough demand, return back on it :)

Cheers :)

**Comment by hargup** [comment]
@federico-terzi Thanks for your work and your comment.
